### ISSUE TITLE
plugin-sdk/v2 is broken

### ISSUE THREAD

**Issue by AlyRagab** [cloudquery/plugin-sdk#888]
### Are you positive this is an SDK bug?

- [X] I'm positive this is an SDK bug

### Current Behavior

Created the project using scaffold command, Scaffold CLI is running with `1.7.0`, Once i want to make sure is fine using `go mod tidy` then `go build` i have the below errors in some of the packages:

```bash
# github.com/cloudquery/plugin-sdk/v2/types
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/inet.go:47:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/inet.go:131:24: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/inet.go:155:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/json.go:49:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/json.go:124:24: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/json.go:150:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/mac.go:43:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/mac.go:125:24: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/mac.go:144:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/uuid.go:35:16: undefined: array.NullValueStr
$HOME/go/pkg/mod/github.com/cloudquery/plugin-sdk/v2@v2.5.1/types/uuid.go:35:16: too many errors
```
Golang Version is : 1.19

### Expected Behavior

It should be built normally using the mentioned version

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by candiduslynx** [comment]
Hi @AlyRagab!
It might seem so, but the issue is caused due to the fact that not all of our upstream commitments to the Apache Arrow were merged yet.

Kindly use the `replace` statement from our [`go.mod`](https://github.com/cloudquery/plugin-sdk/blob/main/go.mod) to use our fork of Apache Arrow.
[This](https://github.com/cloudquery/plugin-sdk/blob/main/go.mod#L29) line, to be exact.

**Comment by hermanschaaf** [comment]
Hi @AlyRagab, thanks for reporting this, and apologies for the issue! We just released a new version of the scaffold tool (v1.8.0) where this issue should be fixed. Could you try upgrading to v1.8.0 and let us know how it goes?

**Comment by hermanschaaf** [comment]
Fixed in Scaffold tool v1.8.0

**Comment by AlyRagab** [comment]
@hermanschaaf thanks, issue is fixed.
